cognite/client/_api/datapoint_tasks.py

```diff
@@ -529,17 +537,17 @@ def __init__(self, *, limit: int, **kwargs: Any) -> None:
         assert isinstance(limit, int)
 
     @property
-    def ts_task_type(self) -> type[ParallelLimitedAggTask]:
-        return ParallelLimitedAggTask
+    def get_task_orchestrator(self) -> type[SerialLimitedAggTaskOrchestrator]:
```

Comment:
just task_orchestrator since it's a property?

Reply:
Aye, agreed